static_src/components/app_page.jsx

```diff
@@ -3,37 +3,47 @@ import React from 'react';
 
 import AppStore from '../stores/app_store.js';
 import Loading from './loading.jsx';
+import OrgStore from '../stores/org_store.js';
+import SpaceStore from '../stores/space_store.js';
 import RouteList from './route_list.jsx';
 
 import sectionStyle from 'cloudgov-style/css/components/section.css';
 import createStyler from '../util/create_styler';
 
+function stateSetter(current) {
+
+  return {
+    app: AppStore.get(current.currentAppGuid) || {},
+    currentAppGuid: current.currentAppGuid,
+    currentOrgName: OrgStore.currentOrgName(),
+    currentSpaceName: SpaceStore.currentSpaceName(),
+    loading: AppStore.fetching
```

Comment:
Now that we have two more stores in here should loading actually be something more like below?

`loading: AppStore.fetching || OrgStore.fetching || SpaceStore.fetching`